### DISCUSSION TITLE
[VB.Net] How to Change User Agent in CefSharp

### DISCUSSION THREAD

**Discussion by Notplayingallday383** [cefsharp/CefSharp#3982] [Q&A]
Hi There, I was wondering how you change the user agent without making some Config Mess that relaunches the app if anybody knows how to do it and would like to share how please do!

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/wiki/General-Usage#useragent

**Reply by Notplayingallday383** [reply]
ok
